### DISCUSSION TITLE
Collapse/Expand Links

### DISCUSSION THREAD

**Discussion by hue1111** [Equim-chan/mjai-reviewer#87] [Ideas]
It would be nice to have some links that can expand and collapse all matches/mistakes/details, to minimize the amount of scrolling or clicking necessary for the user to view the specific content that they would like to see.

**Comment by sakapun** [comment]
@hue1111 
I have made a simple bookmarklet tool outside of this repository.  https://sakapun.github.io/mjai_bookmarklet/  
The instructions is written in Japanese, but it is very simple.  
  
If I have a time, I will be make pull request that can use in this repository.

**Reply by onlca** [reply]
Thanks. It helps a lot.

**Comment by Equim-chan** [comment]
Matched actions are now collapsed by default.
